### DISCUSSION TITLE
GUI cannot display after reboot server.

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1446] [Q&A]
Hi Vincent / Expert,

I reboot the server and after reboot the GUI cannot display. Look like inlet, orchestrator and console stuck as below output. Appreciate your advise how to recover it.

![image](https://github.com/user-attachments/assets/99d38f3d-a528-422e-90a7-149f798a29bf)

root@netflow-akvorado-1:~/akvorado# docker compose up -d
[+] Running 10/0
 ⠿ Container akvorado-traefik-1                   Running                                                                                                  0.0s
 ⠿ Container akvorado-geoip-1                     Running                                                                                                  0.0s
 ⠿ Container akvorado-zookeeper-1                 Running                                                                                                  0.0s
 ⠿ Container akvorado-redis-1                     Running                                                                                                  0.0s
 ⠿ Container akvorado-kafka-1                     Running                                                                                                  0.0s
 ⠿ Container akvorado-clickhouse-1                Running                                                                                                  0.0s
 ⠿ Container akvorado-akvorado-conntrack-fixer-1  Running                                                                                                  0.0s
 ⠿ Container akvorado-akvorado-inlet-1            Started                                                                                                  0.0s
 ⠿ Container akvorado-akvorado-orchestrator-1     Started                                                                                                  0.0s
 ⠿ Container akvorado-akvorado-console-1          Started                                                                                                  0.0s

**Comment by kronicklez** [comment]
Hi @vincentbernat / Expert,

I'm try to restart the docker container but it still fail utk start the docker. Appreciate your feedback. If not have solution look like i need to reinstall back everything.

root@netflow-akvorado-1:~/akvorado# docker compose exec akvorado-orchestrator akvorado orchestrator --check --dump /etc/akvorado/akvorado.yaml
Error response from daemon: Container ca32cfc8ad4011b6e7bc1bda614d57a7f6592b18a08a1000d5e2973816c00f25 is restarting, wait until the container is running
root@netflow-akvorado-1:~/akvorado# docker restart akvorado-akvorado-orchestrator-1
akvorado-akvorado-orchestrator-1
root@netflow-akvorado-1:~/akvorado# 


Thanks

**Reply by vincentbernat** [reply]
I don't know. I can't provide personalized support. Maybe you have something more familiar with system problems. It is likely there are some logs explaining the failure.
